src/subscription_manager/managercli.py

```diff
@@ -973,6 +974,89 @@ def list_service_levels(self):
                 raise e
 
 
+class UsageCommand(OrgCommand):
+
+    def __init__(self):
+
+        shortdesc = _("Manage usage setting for this system")
+        self._org_help_text = _("specify an organization when listing available service levels using the organization key, only used with --list")
+        super(UsageCommand, self).__init__("usage", shortdesc,
+                                                  False)
+
+        self._add_url_options()
+        self.parser.add_option("--show", dest="show", action='store_true',
```

Comment:
If the destination is the same as the "--" word, you [don't need to set it](https://docs.python.org/3/library/argparse.html#dest).  Argparse and Optparse will both do it by default.  You only need `dest` for something like `"--verbose", dest="quiet", action="store_false"` or `"--organization", dest="org"`